smartcontracts/projectregistrar/contracts/projectregistrar.sol

```diff
@@ -9,6 +9,7 @@ contract ProjectRegistrar is ERC721, ReentrancyGuard {
         string uri;
         bytes32 hash;
         bool paused;
+        // operators are not addresses, but uint256 from operator registry
```

Comment:
If we convert the OperatorsRegistry mapping to [profileAddress => {NodeAddress, RewardAddress}] (refer to the comment on the OperatorsRegistry), we can retain this mapping as it currently stands.

However, I believe that the information regarding which project a specific operator wishes to join should be stored in the FleetManagement contract. This is because the FleetManagement logic determines which node(s) are now eligible to submit a proof for a particular project based on a consensus within the network. The initial (centralized) implementation of fleet management logic, essentially involves allowing project owner to specify which operators they wish to permit, then simply verify these mappings when requested by the router.